benchmark_runner/main/main.py

```diff
@@ -206,8 +208,9 @@ def main():
     """
     success = True
     # azure_cluster_start_stop
-    if azure_cluster_stop or azure_cluster_start:
-        azure_cluster_start_stop()
+    if azure_cluster_operation:
+        azure_cluster_operations()
```

Comment:
...in which case this would be
```
azure_cluster_operations(azure_cluster_operation)
```

Reply:
Method: azure_cluster_operations 
azure_cluster_operation: STR Variable START/STOP

Reply:
Right, that should be an argument to `azure_cluster_operation`

Reply:
Update in comments

Reply:
That's not the issue; the value of azure_cluster_operation should be passed as an argument to azure_cluster_operations().

Reply:
Fixed